client/src/partials/cash/modals/transfer.modal.html

```diff
@@ -0,0 +1,88 @@
+<form name="TransferForm" ng-submit="CashTransferModalCtrl.submit(TransferForm.$invalid)" novalidate>
+
+  <div class="modal-header">
+    <h4>{{ "CASH.VOUCHER.CASHBOXES.TRANSFER" | translate }}</h4>
+  </div>
+
+  <div class="modal-body">
+
+    <p class="text-danger text-center" ng-if="CashTransferModalCtrl.cashBoxIdMissed">
+      <span class="glyphicon glyphicon-alert"></span>
+      {{ "CASH.MODALS.MISSING_CASH_BOX_ID" | translate }}
+    </p>
+
+    <uib-alert type="info" ng-if="!CashTransferModalCtrl.cashBoxIdMissed">
+      <p>{{ "CASH.MODALS.FINANCIAL_DETAIL" | translate }} </p>
+
+      <p>{{"CASH.CASHBOX" | translate}} : {{ CashTransferModalCtrl.cashBox.text }} </p>
+      <div ng-if="CashTransferModalCtrl.currency_id">
+        <p>{{"COLUMNS.CURRENCY" | translate}} : {{ CashTransferModalCtrl.currencyService.format(CashTransferModalCtrl.currency_id); }}</p>
+        <p>{{"COLUMNS.BALANCE" | translate}} : {{CashTransferModalCtrl.cashAccountSold.balance}}</p>
```

Comment:
That makes sense.

Is there anywhere that you can get the correct `currency_id` to format this number?